### ISSUE TITLE
Missing titles for reader.annotations.type. in the navigation panel

### ISSUE THREAD

**Issue by gautierchomel** [edrlab/thorium-reader#2700]
The icons for annotation type have titles in the Annotation windows but not in the navigation panel filtering options.

![image](https://github.com/user-attachments/assets/ee2d11cb-d2d2-4bf8-a7fe-7453c1d047ba)
